### PR TITLE
zapier tweaks

### PR DESCRIPTION
**Pull request by czue**
some small tweaks to our zapier integration. can be read by commit. the second commit is actually the only change that really matters.

i'm buddyless! @dannyroberts @proteusvacuum @kaapstorm since this is a little "integrationy"

### CODE REVIEW

File: corehq/apps/zapier/api/v0_5.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def obj_get_list(self, bundle, **kwargs):
             ))
 
         for form_property in MAIN_FORM_TABLE_PROPERTIES:
-            if form_property.is_advanced:
+            if form_property.is_advanced or form_property.item.transform:
```
Comment (czue) [inline corehq/apps/zapier/api/v0_5.py]: this fixes one of the issues highlighted [here](https://cdn.zapier.com/storage/photos/f109159ec8f2cc0cdde9f8ae86a762ab.png) (by removing the field that wasn't getting transformed anyways)

File: corehq/apps/zapier/views.py [hunk 2]
```diff
@@ -69,7 +70,8 @@ def post(self, request, domain, *args, **kwargs):
         else:
             return HttpResponse(status=400)
 
-        return HttpResponse('OK')
+        # respond with the ID so that zapier can use it to unsubscribe
+        return json_response({'id': subscription.id})
```
Comment (dannyroberts) [inline corehq/apps/zapier/views.py]: There's also a django built-in now called JsonRepsonse that that behaves pretty much the same. No need to change now, but that reminds me we should probably get rid of this (now superfluous) util.

**Review by dannyroberts — APPROVED** [review]

File: corehq/apps/zapier/views.py (hunk 2 above)
Comment (dannyroberts) [inline corehq/apps/zapier/views.py]: Actually, I guess ours does use a json handler that calls `to_json()` on an object if possible, so I'll keep it for now.

**Comment by czue** [comment]
I see this is approved. do I merge my own PR now or should I be waiting for something else?

**Comment by proteusvacuum** [comment]
👋 @czue 
You can merge your own branch if it gets approved and tests pass!

**Comment by czue** [comment]
:wave: @proteusvacuum 

> You can merge your own branch if it gets approved and tests pass!

:open_mouth: i don't know if i can handle all this change.

**Pull request merged**